egs/wsj/s5/steps/lmrescore.sh

```diff
@@ -115,7 +115,7 @@ case "$mode" in
     ;;
 esac
 
-rm $outdir/Ldet.fst 2>/dev/null
+rm -f $outdir/Ldet.fst
```

Comment:
In general we avoid the -f flag of 'rm', because people might purposely make a file un-writable to stop it being deleted, and the -f flag overrides that.  Can you please change this to
`rm $outdir/Ldet.fst 2>/dev/null || true`
?